.generator/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "generator"
+version = "0.1.0"
+description = ""
+authors = ["Jiri Kuncar <jiri.kuncar@datadoghq.com>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+click = "8.0.1"
+PyYAML = "6.0"
+jsonref = "0.2"
+jinja2 = "3.0.3"
+
+[tool.poetry.dev-dependencies]
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
```

Comment:
Do we have to use poetry for this? just wondering as we've done without until now.